requirements.txt

```diff
@@ -2,7 +2,7 @@
 # therefore they must be listed here. Moreover, they cannot be listed in
 # setup.py, as pip is not able to install them.
 http://github.com/Supervisor/supervisor/zipball/master#egg=supervisor==4.0.0.dev0
-http://github.com/badochov/djsupervisor/zipball/master#egg=djsupervisor==0.4.0
+http://github.com/LiquidPL/django-supervisor/zipball/master#egg=django-supervisor==0.4.0
```

Comment:
I prefer to have this repo in sio2project organization

Reply:
I've opened a pull request to bring in my changes: sio2project/django-supervisor#1.

Reply:
I've merged your PR to django-supervisor